go.mod

```diff
@@ -40,9 +40,9 @@ require (
 	golang.org/x/sync v0.5.0
 	golang.org/x/text v0.14.0
 	golang.org/x/time v0.3.0
-	google.golang.org/genproto/googleapis/api v0.0.0-20230920204549-e6e6cdab5c13
-	google.golang.org/grpc v1.58.3
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/genproto/googleapis/api v0.0.0-20240125205218-1f4bbc51befe
+	google.golang.org/grpc v1.61.0
+	google.golang.org/protobuf v1.33.0
```

Comment:
Is there a reason to bump protobuf version?

Reply:
hm, good question. I was not specifically looking to bump. But with `ignite` tools installed, I think this was done automatically on generation step. I can roll this back

Reply:
@yzang2019 addressed this